File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.Ex_PB-JS-Conditional-Statements/P08_LunchBreak.js
function lunchBreak(input) {
    let filmName = input[0];
    let filmTime = Number(input[1]);
    let restTime = Number(input[2]);

    let watchTime = restTime * (1 - (1 / 8) - (1 / 4));

    if (watchTime < filmTime) {
        console.log(`You don't have enough time to watch ${filmName}, you need ${Math.ceil(filmTime - watchTime)} more minutes.`);
    } else {
        console.log(`You have enough time to watch ${filmName} and left with ${Math.ceil(watchTime - filmTime)} minutes free time.`)
    }
}

lunchBreak(["Teen Wolf",
    "48",
    "60"])

File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P02Ex_ConditionalStatements/P08_LunchBreak.java
package P02Ex_ConditionalStatements;

import java.util.Scanner;

public class P08_LunchBreak {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String nameOfFilm = scanner.nextLine();
        int durationOfFilm = Integer.parseInt(scanner.nextLine());
        int durationOfRest = Integer.parseInt(scanner.nextLine());

        double time = durationOfRest * (1 - 1 / 8.0 - 1 / 4.0);

        if (time < durationOfFilm)
            System.out.printf("You don't have enough time to watch %s, you need %.0f more minutes.", nameOfFilm,
                    Math.ceil(durationOfFilm - time));
        else
            System.out.printf("You have enough time to watch %s and left with %.0f minutes free time.", nameOfFilm,
                    Math.ceil(time - durationOfFilm));
    }
}
